converter/src/test/java/gov/cms/qpp/conversion/ConverterTest.java

```diff
@@ -74,37 +62,6 @@ public void testValidQppStream() {
 		//no exception should be thrown, hence explicitly stating the expected exception is None
 	}
 
-	@Test
-	@PrepareForTest({Converter.class, QrdaValidator.class})
-	public void testValidationErrors() throws Exception {
```

Comment:
Unless I am missing something, this test still has merit.  We should change it to use a different `TemplateId` and change `JennyDecoder` to be its own decoder instead of extending from `DefaultDecoder`.

Reply:
Okay. I just flat out removed it as we no longer were using defaults. I can see your point. I'll change this.